docs/database/tables/sentences_lists.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE `sentences_lists` (
   `numberOfSentences` mediumint(8) unsigned NOT NULL DEFAULT '0',
   `created` datetime DEFAULT NULL,
   `modified` datetime DEFAULT NULL,
-  `visibility` enum('private', 'unlisted', 'public') NOT NULL DEFAULT 'unlisted',
+  `visibility` enum('private', 'unlisted', 'public', 'public+') NOT NULL DEFAULT 'unlisted',
```

Comment:
Now that you added the migration script, you can remove the public+ from the sql file. This is because we currently initialize the database with these sql scripts, and then run all migrations on the top of that.